src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_ExcelCompat.txt

```diff
@@ -197,4 +191,204 @@ Table({Value:"08:03:05"},{Value:"00:00:00"})
 "1899-12-30"
 
 >> Text(Blank(),"mmm ddd yyy")
-"Dec Sat 1899"
\ No newline at end of file
+"Dec Sat 1899"
+
+>> Text(1234567.1234567, "mmm ddd yyye")
+"Feb Thu 5280e"
+
+>> Text(1234567.1234567, "mmm ddd yyy e")
+"Feb Thu 5280 e"
+
+>> Text(1234567.1234567, "mmm ddd yyy date")
+"Feb Thu 5280 15ate"
+
+>> Text(1234567.1234567, "mmm ddd yyye+")
+"Feb Thu 5280e+"
+
+// DateTimeFormat Enum
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "DateTimeFormat.LongDateTime")
+"15ateTi8eFor8at.Long15ateTi8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'DateTimeFormat.LongDateTime'")
+"'15ateTi8eFor8at.Long15ateTi8e'"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "High'there")
+"2ig2't2ere"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "ZZZ 'longdatetime' ZZZ")
+"ZZZ 'long15ateti8e' ZZZ"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'longdatetime'")
+"'long15ateti8e'"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'longdatetime' ZZZ")
+"'long15ateti8e' ZZZ"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "LongDateTime")
+"Long15ateTi8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "longdatetime")
+"long15ateti8e"
+
+>> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'utc'")
+"'utc'"
```

Comment:
Interestingly, 
``` 
> Text(DateTime(2023, 8, 15, 2, 53, 9, 0), "'utc'")
"'utc'"

> Text(123, "'utc'")
"utc"
```
What happened to the single ticks in the output of the second example?  This isn't a super important case or anything, I'm just worried there is something going on in there that we don't understand.

Reply:
This PR does not ignore ' in numeric format  as we will change in PR: https://github.com/microsoft/Power-Fx/pull/1716. So when this PR and that PR merge, this test result will change to "'utc'".